src/unstract/adapters/ocr/register.py

```diff
@@ -46,4 +44,4 @@ def _build_adapter_list(
                         Common.METADATA: metadata,
                     }
         except ModuleNotFoundError as exception:
-            logger.error(f"Error while importing ocr adapters : {exception}")
+            logger.warning(f"Unable to ocr adapters : {exception}")
```

Comment:
@gaya3-zipstack NIT: Looks like the message is not clear. Shouldn't it be "Unable to import ocr adapters"?

Reply:
Thank you. Yes. Changed now.